.circleci/config.yml

```diff
@@ -8,83 +8,29 @@ defaults:
     # reference syntax)
     working_directory: ~/coq
     docker:
-      - image: ocaml/opam:ubuntu
+      - image: ejgallego/coq_dev
 
   environment: &envvars
-    # required by some of the targets, e.g. compcert, passed for
-    # instance to opam to configure the number of parallel jobs
-    # allowed
-    NJOBS: 2
-    COMPILER: "system"
-    CAMLP5_VER: "6.14"
     NATIVE_COMP: "yes"
 
-    # some useful values
-    TIMING_PACKAGES: &timing-packages "time python"
-
 version: 2
 
 before_script: &before_script
   name: Install system packages
```

Comment:
This name should be changed to "opam switch" or some such thing.

Reply:
As you wish.

Reply:
ping

Reply:
I mean ping